src/settings/toml/tests/deploy_config.rs

```diff
@@ -12,17 +12,6 @@ const PATTERN: &str = "hostname.tld/*";
 const ACCOUNT_ID: &str = "fakeaccountid";
 
 // TOP LEVEL TESTS
-#[test]
-fn it_errors_on_empty_deploy_config() {
```

Comment:
it might make sense to check the warnings in this case, instead of just deleting the tests. I need to figure out how, but I wanted to get this commit up earlier rather than later.

Reply:
Couldn't check for warnings, but I added the tests back into check for HttpRouteDeployConfig::NoRoutes instead.